README.tfdoc.hcl

```diff
@@ -63,25 +50,35 @@ section {
   section {
     title   = "Module Features"
     content = <<-END
-      This module implements the following Terraform resources:
-
-      - `null_resource`
-
-      and supports additional features of the following modules:
-
-      - [mineiros-io/something/google](https://github.com/mineiros-io/terraform-google-something)
+      A [Terraform] module for creating `google_compute_subnetwork` resources
```

Comment:
Maybe we could add the list of resources that the module implements, not sure how much it matters to follow the template, it's just what I've been doing when creating new documentation.

Reply:
you're right! this module was based on https://github.com/mineiros-io/terraform-google-network-subnet so the template used to create it was outdated; I updated it accordingly! thanks for the catch, man!